### PR TITLE
#16 Add CLI for PeopleInSpace

### PR DESCRIPTION
**Pull request by mreichelt**
This is a work in progress (but already public).

### CODE REVIEW

**Comment by joreilly** [comment]
@mreichelt is this still draft or do you think it's in position to be reviewed/merged?

**Comment by mreichelt** [comment]
I think I'll give it a little more code massage - I'd like to add two TODOs to note that some of the code is currently required, but shouldn't be.

File: command-line/src/nativeMain/kotlin/PeopleInSpace.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import com.surrus.common.di.initKoin
+import com.surrus.common.remote.PeopleInSpaceApi
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.runBlocking
+
+fun main() {
```
Comment (LordRaydenMK) [inline command-line/src/nativeMain/kotlin/PeopleInSpace.kt:6]: Nit: you can use `suspend fun main` and remove `runBlocking`

**Comment by joreilly** [comment]
@mreichelt just doing some housekeeping and wondering if you're hoping still to create (not draft) PR for this?  If not, I can try and grab some time and rebase and get across the line.

**Comment by RdeWilde** [comment]
@mreichelt did you ever finish this?

**Pull request closed without merging**